DESCRIPTION

```diff
@@ -114,7 +114,8 @@ VignetteBuilder:
     knitr
 Encoding: UTF-8
 RoxygenNote: 7.2.3
-Config/testhat/edition: 3
+Roxygen: list(markdown = TRUE)
+Config/testthat/edition: 3
```

Comment:
```suggestion
Config/testthat/edition: 2
```

As the failures are unrelated to the main purpose of this PR, maybe making this change would be sufficient to reduce warnings in tests, given that you agree with the other changes made.